docs/content/en/docs/reference/packagespec/prometheus/v2.39.1.md

```diff
@@ -0,0 +1,565 @@
+| nodeExporter.service | Specifies how to expose node-exporter as a network service | See footnote [^1] |
+| nodeExporter.tolerations | Specifies node tolerations for node-exporter scheduling to nodes with taints. Refer to the Kubernetes API documentation [toleration](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#toleration-v1-core) field for more details. | See footnote [^2] |
+| serviceAccounts.nodeExporter.annotations | Specifies node-exporter service account annotations | `{}` |
+| serviceAccounts.nodeExporter.create | Indicates if node-exporter service account will be created | `true` |
+| serviceAccounts.nodeExporter.name | Specifies node-exporter service account name | `""` |
+|**Prometheus-Server**|
+| server.enabled | Indicates if prometheus-server is enabled | `true` |
+| server.global.evaluation_interval | Specifies how frequently the prometheus-server rules are evaluated | `"1m"` |
+| server.global.scrape_interval | Specifies how frequently prometheus-server will scrape targets | `"1m"` |
+| server.global.scrape_timeout | Specifies how long until a prometheus-server scrape request times out | `"10s"` |
+| server.image.pullPolicy | Specifies prometheus-server image pull policy: `IfNotPresent`, `Always`, `Never` |`"IfNotPresent"` |
+| server.image.repository | Specifies prometheus-server image repository | `"prometheus/prometheus"` |
+| server.name | Specifies prometheus-server container name | `"server"` |
+| server.persistentVolume.accessModes | Specifies prometheus-server data Persistent Volume access modes | `"ReadWriteOnce"` |
+| server.persistentVolume.enabled | Indicates if prometheus-server will create/use a Persistent Volume Claim | `true` |
+| server.persistentVolume.existingClaim | Specifies prometheus-server data Persistent Volume existing claim name. It requires `server.persistentVolume.enabled: true`. If defined, PVC must be created manually before volume will be bound | `""` |
+| server.persistentVolume.size | Specifies prometheus-server data Persistent Volume size | `"8Gi"` |
+| server.remoteRead | Specifies prometheus-server remote read configs. Refer to Prometheus docs [remote_read](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_read) for more details | `[]` |
+| server.remoteRead | Specifies prometheus-server remote write configs. Refer to Prometheus docs [remote_write](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write) for more details | `[]` |
+| server.replicaCount | Specifies the replicaCount for prometheus-server deployment / statefulSet. **Note `server.statefulSet.enabled` should be set to `true` if `server.replicaCount` is greater than `1`**  | `1` |
```

Comment:
```suggestion
| server.replicaCount | Specifies the replicaCount for prometheus-server deployment / statefulSet. **Note:** `server.statefulSet.enabled` should be set to `true` if `server.replicaCount` is greater than `1`  | `1` |
```
Not required, but I like to just bold the "Note" part.

Reply:
Updated.